backend/tilavarauspalvelu/integrations/email/template_context/reservation.py

```diff
@@ -404,22 +405,32 @@ def get_context_for_reservation_requires_payment(
 ) -> Annotated[EmailContext, EmailType.RESERVATION_REQUIRES_PAYMENT]:
     if reservation is not None:
         data["email_recipient_name"] = reservation.actions.get_email_reservee_name()
-        data["payment_due_date"] = local_date()
         data["instructions_confirmed"] = reservation.actions.get_instructions(kind="confirmed", language=language)
 
+        # TODO: Get from payment order handled payment due by
+        data["handled_payment_due_by"] = reservation.begin - datetime.timedelta(hours=1)
```

Comment:
This is wrong, but because this email changes needs to be both on top of the email changes made for the access code changes, but also requires the changes fr handled payments, `handled_payment_due_by` doesn't exist yet on this branch.

Reply:
Added this PR to the current merge group **5** along with the other email changes. I guess that group could be moved up in the queue (swapped around with **4**) to facilitate further development.

Reply:
This should now be fixed